app/controllers/category_expenditures_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CategoryExpendituresController < ApplicationController
+  before_action :set_expenditure_category, only: %i[show edit update destroy]
+
+  before_action :authenticate_user!
+
+  def index
+    @category = Category.find(params[:category_id])
+    @expenditures = @category.category_expenditures.includes(:expenditure).map(&:expenditure)
+    # order expenditures by most recent
+    @expenditures = @expenditures.sort_by(&:created_at).reverse
+  end
+end
```

Comment:
- [ ] also you should add a back button to navigate back so please try to add it in order to match the requirements 
- 
<img width="1004" alt="Screenshot 2022-11-17 at 13 44 49" src="https://user-images.githubusercontent.com/61790587/202449973-2376dcfd-0937-4846-ad85-054e27e8554c.png">
- Yours 
![Uploading Screenshot 2022-11-17 at 13.15.43.png…]()

Reply:
I have added the back button in the new transaction page